doc-content/shared-kie-docs/src/main/asciidoc/KieServerTaskAssigning/Examples-section.adoc

```diff
@@ -139,7 +139,7 @@ image::KieServerTaskAssigning/StartCreditDisputeInstance3Simple.png[]
 .Resolve dispute task for process instance #3
 image::KieServerTaskAssigning/ResolveDisputeTaskForProcessInstance3WasCreated.png[]
 
-The ResolveDispute task is assigned to the user katy because that task requires MASTER skills.
+The ResolveDispute task is assigned to the user katy because that task requires skills in MASTER card.
```

Comment:
```suggestion
The ResolveDispute task is assigned to the user Katy because that task requires skills in the MASTER card.
```

Reply:
@kaldesai This statement refers to the following code snippet that is available in a previous section:

john=ES,EN                     (has skills in the Spanish and English languages)
mary=ZH                        (has skills in the Chinese language)
katy=CITI,MASTER               (has skills in the CITI and MASTER cards)
maciek=VISA,AMERICAN_EXPRESS   (has skills in the VISA and AMERICAN EXPRESS cards)

This snippet has the katy with small k, therefore I'll keep it as is, and adding the article `the` in the sentence as suggested. 
Thank you for your review